### PR TITLE
Add basic tests for components

### PR DESCRIPTION
**Pull request by bastilian**

### CODE REVIEW

File: src/PresentationalComponents/NotificationActions/NotificationActions.js [hunk 1]
```diff
@@ -8,9 +8,9 @@ import {
     DropdownItem
 } from '@patternfly/react-core';
 
-class NotificationActions extends React.Component {
+export class NotificationActions extends React.Component {
     state = {
-        isOpen: this.props.isOpen
+        isOpen: (this.props.isOpen === undefined) ? false : this.props.isOpen
```
Comment (adamruzicka) [inline src/PresentationalComponents/NotificationActions/NotificationActions.js]: Doesn't `undefined` count as a falsey value?

File: config/setupTests.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+import fetch from 'cross-fetch';
 
 configure({ adapter: new Adapter() });
 
+global.fetch = fetch;
 global.shallow = shallow;
 global.render = render;
 global.mount = mount;
 global.React = React;
+global.insights = {};
```
Comment (adamruzicka) [inline config/setupTests.js]: Why is this needed?

File: src/SmartComponents/NotificationEdit/NotificationEdit.js [hunk 3]
```diff
@@ -87,7 +86,8 @@ NotificationEdit.propTypes = {
     endpointId: PropTypes.number,
     endpoint: PropTypes.object,
     fetchEndpoint: PropTypes.func.isRequired,
-    match: PropTypes.object
+    match: PropTypes.object,
+    store: PropTypes.object
```
Comment (adamruzicka) [inline src/SmartComponents/NotificationEdit/NotificationEdit.js]: Is this so we can inject a fake store for testing?

File: src/PresentationalComponents/NotificationActions/NotificationActions.js [hunk 4]
```diff
@@ -8,9 +8,9 @@ import {
     DropdownItem
 } from '@patternfly/react-core';
 
-class NotificationActions extends React.Component {
+export class NotificationActions extends React.Component {
```
Comment (adamruzicka) [inline src/PresentationalComponents/NotificationActions/NotificationActions.js:11]: I assume all these `export`s are to be able to use the not-yet-connected-to-store component in the tests, correct?

**Review by adamruzicka — COMMENTED** [review]
Looks good to me and Travis seems to be satisfied, left some inline comments just to make sure I follow what's going on

File: config/setupTests.js (hunk 2 above)
Comment (bastilian) [reply config/setupTests.js]: This was overlooked, it isn't needed, but seemed to be at somepoint when getting the tests to run...

File: src/PresentationalComponents/NotificationActions/NotificationActions.js (hunk 1 above)
Comment (bastilian) [reply src/PresentationalComponents/NotificationActions/NotificationActions.js]: indeed, it is.

File: src/SmartComponents/NotificationEdit/NotificationEdit.js (hunk 3 above)
Comment (bastilian) [reply src/SmartComponents/NotificationEdit/NotificationEdit.js]: another leftover from trying to get tests work.

File: src/PresentationalComponents/NotificationActions/NotificationActions.js (hunk 4 above)
Comment (bastilian) [reply src/PresentationalComponents/NotificationActions/NotificationActions.js:11]: Correct. :)

File: src/SmartComponents/NotificationsIndex/NotificationIndex.test.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    it('expect to render a Table', () => {
+        const wrapper = shallow(
+            <NotificationsIndex { ...defaultProps }/>
+        );
+        expect(wrapper.find('Table').length).toBe(1);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it('it takes endpoints', () => {
```
Comment (adamruzicka) [inline src/SmartComponents/NotificationsIndex/NotificationIndex.test.js]: Too many `it`s

**Review by adamruzicka — COMMENTED** [review]
One last nitpick, then it's good to go

File: src/SmartComponents/NotificationsIndex/NotificationIndex.test.js (hunk 5 above)
Comment (bastilian) [reply src/SmartComponents/NotificationsIndex/NotificationIndex.test.js]: Removed `it`.

**Comment by adamruzicka** [comment]
Thanks @bastilian !
